hw/xbox/nv2a/nv2a.h

```diff
@@ -21,6 +21,24 @@
 #ifndef HW_NV2A_H
 #define HW_NV2A_H
 
+/*These types are declared here, since nv2a.h is a common library and removes the need of 
```

Comment:
It's fine as a note while you're working on this, but this isn't something that should stay in the final PR.